### PR TITLE
[FUSEQE-1425] Fix salesforce + retry login when unsuccessful

### PR DESCRIPTION
**Pull request by avano**
//test: @integrations-tw-sf

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/SfValidationSteps.java [hunk 1]
```diff
@@ -35,19 +35,32 @@
...
     public SfValidationSteps() {
         accountsDirectory = AccountsDirectory.getInstance();
         final Account salesforceAccount = accountsDirectory.getAccount("QE Salesforce").get();
-        salesforce = new ForceApi(new ApiConfig()
-                .setClientId(salesforceAccount.getProperty("clientId"))
-                .setClientSecret(salesforceAccount.getProperty("clientSecret"))
-                .setUsername(salesforceAccount.getProperty("userName"))
-                .setPassword(salesforceAccount.getProperty("password"))
-                .setForceURL(salesforceAccount.getProperty("loginUrl")));
+        int retries = 0;
```
Comment (tplevko) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/SfValidationSteps.java]: How often is there actually this connection issue?

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**